src/PerfCounters_x86.h

```diff
@@ -74,16 +75,16 @@ static CpuMicroarch compute_cpu_microarch() {
 	return IntelCometlake;
     case 0x30f00:
       return AMDF15R30;
-    case 0x00f10:
+    case 0x00f10: // Naples, Whitehaven, Summit Ridge, Snowy Owl (Zen)
```

Comment:
Until now we've basically only added lines for archs we have actually tested. I think it's OK to add lines for CPUs we expect to work, but I still want to record which ones have (not) been tested. So please add `(UNTESTED)` for every CPU other than yours that you're adding here.

Reply:
After this has landed we will put out a call for volunteers to test the other microarchs.